lib/rubocop/cop/style/round.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for the use of the round method.
```

Comment:
The documentation here should be significantly extended and include code examples. See other cops for reference.